rust/rope/src/delta.rs

```diff
@@ -22,10 +22,16 @@ use subset::{Subset, SubsetBuilder};
 use std::cmp::min;
 
 enum DeltaElement<N: NodeInfo> {
+    /// Represents the text in a range of the base document. Includes beginning, excludes end.
```

Comment:
maybe "a range in the base document", or "a range of text in the base document".